spec/components/section_spec.rb

```diff
@@ -8,6 +8,24 @@ class ExampleSection < Ninetails::Section
 
 RSpec.describe Ninetails::Section do
 
+  describe "creating a section from a file name" do
+    it "should handle plural names" do
+      expect(Ninetails::Section.new_from_filename("points")).to be_a Section::Points
+    end
+
+    it "should handle singular names" do
+      expect(Ninetails::Section.new_from_filename("billboard")).to be_a Section::Billboard
+    end
+
+    it "should handle names with suffices" do
+      expect(Ninetails::Section.new_from_filename("billboard.rb")).to be_a Section::Billboard
+    end
+
+    it "should handle absolute paths" do
+      expect(Ninetails::Section.new_from_filename("/foo/bar/billboard.rb")).to be_a Section::Billboard
+    end
+  end
```

Comment:
Yeah, kinda, but it's testing that it instantiates the correct `Section`, so I'm happy..